apps/sms/views/conversation/js/compose.js

```diff
@@ -435,7 +433,7 @@ var Compose = (function() {
       if (message.type === 'mms') {
         if (message.subject) {
           this.setSubject(message.subject);
-          this.showSubject();
+          subject.show();
```

Comment:
same than https://github.com/mozilla-b2g/gaia/pull/31055/files#r35208531
(and this code will disappear in https://github.com/mozilla-b2g/gaia/pull/30630 anyway)